web/resources/js/hello.ts

```diff
@@ -0,0 +1,7 @@
+import $ from "jquery";
```

Comment:
Remove this commit from the PR? With `generic-typeahead.ts` there seems to be a proper typescript module now.